addons/explosives/CfgVehicles.hpp

```diff
@@ -279,38 +279,41 @@ class CfgVehicles {
 
     // Add ACE_Clacker and ACE_DefusalKit to every explosive unit.
     #define MACRO_ADDMINEKIT \
-    items[] = {"FirstAidKit", "ACE_Clacker", "ACE_DefusalKit"}; \
-    respawnitems[] = {"FirstAidKit", "ACE_Clacker", "ACE_DefusalKit"};
+    items[] = {"FirstAidKit","ACE_Clacker","ACE_DefusalKit"}; \
+    respawnitems[] = {"FirstAidKit","ACE_Clacker","ACE_DefusalKit"};
```

Comment:
Unnecessary changes, unless we decide on spacing standard. Changes like that just become merge conflicts.